File: Day13/Solution.java
class MyBook extends Book{
    private int book_Price;
     MyBook(String title, String author, int price){
         super(title, author);
         this.book_Price = price;
     }
    public void display(){
        System.out.println("Title: "+title);
        System.out.println("Author: "+author);
        System.out.println("Price: "+book_Price);
    }
}

File: Day13/Solution.cpp
class MyBook:public Book {
    
    int price;
    public:
        MyBook(string title, string author, int p):Book(title, author) {
            price = p;
        }

        void display() {
            cout<<"Title: "<<title<<endl;
            cout<<"Author: "<<author<<endl;
            cout<<"Price: "<<price<<endl;
        }
};
